tests/e2e/scenarios/test_reborn_webui_v2_smoke.py

```
@@ -933,6 +933,59 @@ async def test_reborn_v2_thread_list_and_delete(reborn_v2_server):
+    headers = {"Authorization": f"Bearer {REBORN_V2_AUTH_TOKEN}"}
+    async with httpx.AsyncClient(headers=headers) as client:
+        keep_id = await _create_thread(client, reborn_v2_server)
+        drop_id = await _create_thread(client, reborn_v2_server)
+
+    context = await reborn_v2_browser.new_context(viewport={"width": 1280, "height": 720})
+    page = await context.new_page()
+    release_refetch = asyncio.Event()
+    refetch_started = asyncio.Event()
+    refetch_finished = asyncio.Event()
+
+    async def delay_thread_list_refetch(route, _request):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `delay_thread_list_refetch`

Add return type annotation: `None`

(ANN202)